src/ansys/dpf/post/misc_results.py

```diff
@@ -105,74 +109,75 @@ def _check_nodal_location(self, **kwargs):
 
 
 class MecanicMisc(Misc):
+    """Miscellaneous mechanical result."""
 
     # nodal results
     def nodal_displacement(self, **kwargs):
-        """Returns a nodal_displacement result data."""
+        """Return a nodal_displacement result data."""
```

Comment:
```suggestion
        """Get result data for the nodal_displacement."""
```

Reply:
@akaszynski For all methods where I've begun the description with "Get result data for...", I'm wondering if it simply say Get the ...." For example, the description in line 116 might only need to be "Get the nodal replacement." Wouldn't a developer know that a ``ResultData`` object would be returned?

Reply:
If we had a returns section they would.